airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json

```diff
@@ -18538,7 +18538,7 @@
     "sourceDefinitionId": "435bb9a5-7887-4809-aa58-28c27df0d7ad",
     "name": "MySQL",
     "dockerRepository": "airbyte/source-mysql",
-    "dockerImageTag": "2.0.21",
+    "dockerImageTag": "2.0.22",
```

Comment:
No longer need to do this with publish on merge (I don't think file exists anymore)